lib/src/main/java/org/team100/lib/localization/NotePosition24ArrayListener.java

```diff
@@ -54,38 +58,33 @@ void consumeValues(NetworkTableEvent e) {
     }
 
     /**
-     * @return The x position in the camera in pixels, 0 should be the left of the
-     *         screen
-     */
-    public Optional<Double> getX() {
-        switch (Identity.instance) {
-            case BETA_BOT:
-                double xd = positions[0].getX();
-                Double x = xd;
-                Optional<Double> e = Optional.of(x);
-                return e;
-            default:
-                return Optional.empty();
+     * @return The y position in the camera in pixels, 0 should be the bottom of the screen 
+    */
+    public Optional<Translation2d>[] getTranslation2d() {
+        Optional<Translation2d>[] notes = new Optional[]{Optional.empty()};
+        int noteNum = 0;
+        Optional<Double> y = Optional.empty();
+        Optional<Double> x = Optional.empty();
+        Transform3d cameraInRobotCoordinates = Camera.get(id).getOffset();
+        CameraAngles ed = new CameraAngles(cameraInRobotCoordinates);
+        for (NotePosition24 position : positions) {
+            if (position.getPitch() < Math.PI / 2 - cameraInRobotCoordinates.getRotation().getY()) {
```

Comment:
i'm confused about this logic.  isn't the pitch measured from straight ahead?  how does this inequality represent the horizon?

Reply:
It would be facing downward if the camera pitch object is zero

Reply:
in what coordinates?  what is the pitch measuring?